src/test/resources/feature/WriteReview.feature

```diff
@@ -0,0 +1,17 @@
+@Irina
+Feature:   Write Review
+
+  Scenario: WriteReview - Positive
+    Given user is logged in
+    And "Home" is displayed
+    And user clicks on "item1"
+    And "item1's detailed page" is displayed
+    And user clicks on "Write a review"
+    And "WRITE A REVIEW pop-up" is displayed
+    And user clicks on "Quality 3 stars"
+    And user clicks on "Title field"
+    And user enters "VALID DATA"
```

Comment:
Can we have invalid data for Title field?

Reply:
That would be blank value, for ex.